mobile/src/main/java/com/gelakinetic/mtgfam/helpers/database/CardDbAdapter.java

```diff
@@ -699,6 +729,61 @@ public static long fetchIdByName(String name, SQLiteDatabase mDb) throws Familia
         }
     }
 
+    /**
+     * Given a card name, return the KEY_ID of matching cards, searching every language.
+     * <p>
+     * TODO online only pref
+     *
+     * @param name The name of the card
+     * @param mDb  The database to query
+     * @return The KEY_ID values
+     * @throws FamiliarDbException If something goes wrong
+     */
+    public static long[] fetchIdsByLocalizedName(String name, SQLiteDatabase mDb) throws FamiliarDbException {
+        /* replace lowercase ae with Ae */
+        name = sanitizeString(name, true);
+
+        String sql = "SELECT " +
+                DATABASE_TABLE_CARDS + "." + KEY_ID +
+                " FROM " + DATABASE_TABLE_CARDS +
+                " JOIN " + DATABASE_TABLE_SETS +
+                " ON " + DATABASE_TABLE_CARDS + "." + KEY_SET + "=" +
+                DATABASE_TABLE_SETS + "." + KEY_CODE +
+                " WHERE " + name + " COLLATE NOCASE IN (" +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_CHINESE_TRADITIONAL + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_CHINESE_SIMPLIFIED + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_FRENCH + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_GERMAN + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_ITALIAN + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_JAPANESE + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_PORTUGUESE_BRAZIL + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_RUSSIAN + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_SPANISH + "," +
+                DATABASE_TABLE_CARDS + "." + KEY_NAME_NO_ACCENT_KOREAN +
+                ") ORDER BY " + DATABASE_TABLE_SETS + "." + KEY_DATE + " DESC";
+
+        Cursor cursor = null;
+        try {
+            cursor = mDb.rawQuery(sql, null);
+            if (cursor != null && cursor.getCount() > 0) {
+                long[] ids = new long[cursor.getCount()];
+                for (int i=0; i<cursor.getCount(); i++) {
+                    cursor.moveToNext();
+                    ids[i] = cursor.getLong(cursor.getColumnIndex(CardDbAdapter.KEY_ID));
+                }
+                return ids;
+            }
+            return new long[0];

```

Comment:
Did you test what happens when this gets returned?

Reply:
From the gatherer URI intent, a "Not found" Exception is thrown